oneview_redfish_toolkit/api/computer_system.py

```diff
@@ -15,10 +15,36 @@
 # under the License.
 
 import collections
+
+from oneview_redfish_toolkit.api.errors import OneViewRedfishError
 from oneview_redfish_toolkit.api.redfish_json_validator \
     import RedfishJsonValidator
 
 
+POWER_STATE_MAP = {
+    "On": {
+        "powerState": "On",
+    },
+    "ForceOff": {
+        "powerState": "Off",
+        "powerControl": "PressAndHold"
+    },
+    "GracefulShutdown": {
+        "powerState": "Off",
+        "powerControl": "MomentaryPress"
+    },
+    "GracefulRestart": {
+        "powerState": "On"
+    },
+    "ForceRestart": {
+        "powerState": "On"
```

Comment:
The same as above but with powerControl = "ColdBoot"; we did discussed this in the channel... what happened?

Reply:
Daniel said: "Just a general comment on the mappings below, in OV when doing an 'On' operation you cannot also supply a powerControl value. I believe powerControl value is only used for 'Off' operations.". So, I changed it. But I'll fixed it again. Thanks!

Reply:
I tried each combination in OneView and I saw that:
* when the SH is powered on, and you send `{ "powerState": "On"}` (redfish equivalent for `GracefulRestart`) it does not restart; but when you send `{ "powerState": "On", "powerControl": "Reset"}` it indeed restarted.
And the same happened with `{ "powerState": "On", "powerControl": "ColdBoot"}`.

Is there any other way of performing this reset operation, @danielreed?

Reply:
My bad, sorry guys, Ricardo is right about those operations working if the power is already On.  It won't work if the power is Off.

Reply:
Yep! I tested this case too: with a powered off SH the reset operation fails as expected.

Reply:
No problem.